packages/clay-data-provider/src/useResource.ts

```diff
@@ -193,7 +193,7 @@ const useResource = ({
 
 				promise = fn(getUrlFormat(link, variables), fetchOptions).then(
 					(res: Response) => {
-						if (res.ok && res.body && !res.body.locked) {
+						if (res.ok && !res.bodyUsed) {
```

Comment:
Strangely, MDN [claims IE doesn't support `bodyUsed`](https://developer.mozilla.org/en-US/docs/Web/API/Body/bodyUsed), but I guess you tested it? (They also label it as "Experimental. Expect behavior to change in the future.", but [they do that for `body` too](https://developer.mozilla.org/en-US/docs/Web/API/Body/body)). 🤷

Reply:
Hey @wincent yeah, I had also noticed this, am I guessing that there might be some polyfill on the portal? I only see the `bodyUsed` when I was testing it in IE 11.